<launch>
    
  <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen">

    <param name="resolution" value="0.25" />

    <param name="frame_id" type="string" value="world" />
    <param name="sensor_model/sensor_frame_id" type="string" value="camera_link" />
    <param name="sensor_model/max_range" value="10" />

    <remap from="cloud_in" to="/orb_slam3/tracked_points" />
    <!-- Вероятность "попадания" (hit) - насколько доверять точке как препятствию -->
    <param name="sensor_model/hit" value="0.7" /> 
    <!-- Вероятность "промаха" (miss) - насколько доверять лучу как свободному пространству -->
    <param name="sensor_model/miss" value="0.3" />q
    <!-- Минимальная/максимальная вероятность для ячейки (для предотвращения абсолютной уверенности) -->
    <param name="sensor_model/min" value="0.12" />
    <param name="sensor_model/max" value="0.97" />

    <param name="publish_map_topic" value="false"/>
    <param name="map_topic" value="/projected_map"/> 

    <param name="latch" value="false"/>

    <param name="publish_free_space" type="bool" value="true" />
  </node>

</launch>

    
